2015/proposals/banaszkiewicz-piotr-amy.md

```diff
@@ -0,0 +1,214 @@
+# Enhance Amy, a workshop-management platform for Software Carpentry
+
+## Abstract
+
+The number of Software Carpentry's workshops run weekly dynamically grows. All
```

Comment:
On Tue, Mar 24, 2015 at 01:20:29PM -0700, Piotr Banaszkiewicz wrote:
> I've just got data from Greg, I'm producing an updated chart. Not
> sure where to publish it, though.

He'd probably like a “look at how busy we are” blog post on
swcarpentry/site.